flight/targets/omnibusf3/board-info/board_hw_defs.c

```diff
@@ -0,0 +1,977 @@
+		.remap = GPIO_AF_9,
+		.pin = {
+			.gpio = GPIOA,
+			.init = {
+				.GPIO_Pin = GPIO_Pin_2,
+				.GPIO_Speed = GPIO_Speed_2MHz,
+				.GPIO_Mode  = GPIO_Mode_AF,
+				.GPIO_OType = GPIO_OType_PP,
+				.GPIO_PuPd  = GPIO_PuPd_UP
+			},
+			.pin_source = GPIO_PinSource2,
+		},
+	},
+	// To consider
+	// Ch5 - TIM2_CH3  (PB10) -- shared with UART3
+	// Ch6 - TIM2_CH4  (PB11) -- shared with UART3
+	// Ch7 - TIM4_CH2  (PB7)  -- shared with I2C
+	// Ch8 - TIM4_CH1  (PB6   -- shared with I2C
+};
+
+
+#if defined(PIOS_INCLUDE_SERVO) && defined(PIOS_INCLUDE_TIM)
+/*
+ * Servo outputs
+ */
+#include <pios_servo_priv.h>
+
+static const struct pios_servo_cfg pios_servo_cfg = {
+	.tim_oc_init = {
+		.TIM_OCMode = TIM_OCMode_PWM1,
+		.TIM_OutputState = TIM_OutputState_Enable,
+		.TIM_OutputNState = TIM_OutputNState_Disable,
+		.TIM_Pulse = PIOS_SERVOS_INITIAL_POSITION,
+		.TIM_OCPolarity = TIM_OCPolarity_High,
+		.TIM_OCNPolarity = TIM_OCPolarity_High,
+		.TIM_OCIdleState = TIM_OCIdleState_Reset,
+		.TIM_OCNIdleState = TIM_OCNIdleState_Reset,
+	},
+	.channels = pios_tim_servoport_pins,
+	.num_channels = NELEMENTS(pios_tim_servoport_pins),
+};
+
+struct pios_servo_cfg pios_servo_slow_cfg = {
+	.tim_oc_init = {
+		.TIM_OCMode = TIM_OCMode_PWM1,
+		.TIM_OutputState = TIM_OutputState_Enable,
+		.TIM_OutputNState = TIM_OutputNState_Disable,
+		.TIM_Pulse = PIOS_SERVOS_INITIAL_POSITION,
+		.TIM_OCPolarity = TIM_OCPolarity_High,
+		.TIM_OCNPolarity = TIM_OCPolarity_High,
+		.TIM_OCIdleState = TIM_OCIdleState_Reset,
+		.TIM_OCNIdleState = TIM_OCNIdleState_Reset,
+	},
+	.force_1MHz = true,
+	.channels = pios_tim_servoport_pins,
+	.num_channels = NELEMENTS(pios_tim_servoport_pins),
+};
+
+#endif	/* PIOS_INCLUDE_SERVO && PIOS_INCLUDE_TIM */
+
+
+
+/*
+ * PPM Input
+ */
+#if defined(PIOS_INCLUDE_PPM)
+#include <pios_pwm_priv.h>
+
+/*
+ * 	INPUT - PPM
+ *
+ * 	TIM3 CH1 PB4
+ */
+static const struct pios_tim_channel pios_tim_rxport_ppm[] = {
+	{
+		.timer = TIM3,
+		.timer_chan = TIM_Channel_1,
+		.remap = GPIO_AF_2,
+		.pin = {
+			.gpio = GPIOB,
+			.init = {
+				.GPIO_Pin = GPIO_Pin_4,
+				.GPIO_Speed = GPIO_Speed_2MHz,
+				.GPIO_Mode  = GPIO_Mode_AF,
+				.GPIO_OType = GPIO_OType_PP,
+				.GPIO_PuPd  = GPIO_PuPd_UP
+			},
+			.pin_source = GPIO_PinSource4,
+		},
+	},
+};
+#endif
+
+/*
+ * PPM Input
+ */
+#if defined(PIOS_INCLUDE_PPM)
+#include <pios_ppm_priv.h>
+static const struct pios_ppm_cfg pios_ppm_cfg = {
+	.tim_ic_init = {
+		.TIM_ICPolarity = TIM_ICPolarity_Rising,
+		.TIM_ICSelection = TIM_ICSelection_DirectTI,
+		.TIM_ICPrescaler = TIM_ICPSC_DIV1,
+		.TIM_ICFilter = 0x0,
+		.TIM_Channel = TIM_Channel_1,
+	},
+	/* Use only the first channel for ppm */
+	.channels = pios_tim_rxport_ppm,
+	.num_channels = 1,
+};
+
+#endif //PPM
+
+#if defined(PIOS_INCLUDE_ADC)
+#include "pios_adc_priv.h"
+#include "pios_internal_adc_priv.h"
+
+//-------------------------
+// ADC
+// ADC0 : PA0 ADC1_IN1  VBat
+// ADC1 : PA1 ADC1_IN2  Current
+// ADC2 : PB1 ADC3_IN1 RSSI
```

Comment:
Not included at this stage?

Reply:
No, not included.  Nor does betaflight support it, but the "original" board seems to have it.  Will comment